llms/mlx_lm/utils.py

```diff
@@ -134,12 +140,14 @@ def generate(
     return tokens
 
 
-def load(path_or_hf_repo: str) -> Tuple[nn.Module, PreTrainedTokenizer]:
+def load(path_or_hf_repo: str, **kwargs) -> Tuple[nn.Module, PreTrainedTokenizer]:
```

Comment:
Personally I would make this an explicit keyword argument called `tokenizer_config`. If they start to become a lot we can refactor to kwargs dict.

Reply:
Yeah, I was thinking of making it a bit flexible. If we find that the usage of kwargs is getting out of control, we can always extract it from kwargs without breaking the contract. But I am happy to go either way.